sdk/storage/azure-storage-blob/src/main/java/com/azure/storage/blob/options/BlobParallelUploadOptions.java

```diff
@@ -1,22 +1,58 @@
 // Copyright (c) Microsoft Corporation. All rights reserved.
 // Licensed under the MIT License.
 
-package com.azure.storage.blob.models;
+package com.azure.storage.blob.options;
 
 import com.azure.core.annotation.Fluent;
+import com.azure.storage.blob.models.AccessTier;
+import com.azure.storage.blob.models.BlobHttpHeaders;
+import com.azure.storage.blob.models.BlobRequestConditions;
+import com.azure.storage.blob.models.ParallelTransferOptions;
+import reactor.core.publisher.Flux;
 
+import java.io.InputStream;
+import java.nio.ByteBuffer;
+import java.time.Duration;
 import java.util.Map;
 
 /**
  * Extended options that may be passed when uploading a Block Blob in parallel.
  */
 @Fluent
 public class BlobParallelUploadOptions {
+    private final InputStream dataStream;
```

Comment:
This should include `Flux<ByteBuffer>` as well and a ctor overload for it.

Reply:
See response to Alan's comment. Resolving per offline discussion